src/main/resources/cms.conf

```diff
@@ -196,3 +196,10 @@ cms.iam.token.ttl=1h
 #   When true, if an SDB grants access to AD group 'Lst-foo', then users in group 'Lst-Foo' will not have access
 #   When false, if an SDB grants access to AD group 'Lst-foo', then users in group 'Lst-Foo' will have access
 cms.user.groups.caseSensitive=true
+
+cms.encryption.encrypt.cache.maxSize=0
+cms.encryption.encrypt.cache.maxAge=5
```

Comment:
I think we were going to have it default to 0 here, basically disabling caching and then manually enable it via the Cerberus Lifecycle cli.

Reply:
I set it to 5 because the constructor doesn't like 0. I can throw an if statement in front of it but since the cache size is set to 0 there will be no caching either way